src/__tests__/store/reducers/userSession.reducer.test.js

```diff
@@ -1,21 +1,59 @@
-import userSessionReducer from '../../../store/reducers/userSession.reducer';
+import userSessionReducer, { initialState } from '../../../store/reducers/userSession.reducer';
```

Comment:
Same as the other PR. You may not need to export the `initialState` variable in the reducer file if you use something else for your tests.

Reply:
Yeap 😞